tests/src/test/java/alluxio/server/ft/journal/TriggeredCheckpointTest.java

```diff
@@ -107,9 +107,9 @@ private void createFiles(MultiProcessCluster cluster, int numFiles)
     for (int i = 0; i < numFiles; i++) {
       fs.createFile(new AlluxioURI("/file" + i)).close();
     }
-    MetaMasterClient meta = cluster.getMetaMasterClient();
-    assertEquals(numFiles + 1,
-        meta.getMetrics().get("Master." + MasterMetrics.TOTAL_PATHS).getLongValue());
+    MetricsMasterClient metricsMasterClient = cluster.getMetricsMasterClient();
+    assertEquals(numFiles + 1, (long) metricsMasterClient
+        .getMetrics().get("Master." + MasterMetrics.TOTAL_PATHS).getDoubleValue());
```

Comment:
is the cast here correct? Or should we make `numFiles + 1` a double (applies to all cases in tests)

Reply:
The reason that I make `double -> long` is that `assertEquals(double,double)` is deprecated and may have some problems in double comparison.